### ISSUE TITLE
Failed to import from PlexWatch

### ISSUE THREAD

**Issue by bavocorteville** [Tautulli/Tautulli#764]
**Version:**
v1.4.6

**Branch:**
master

**Commit hash:**
d9474cdcc5e72ff311569a234e3abe7dec06d9a2

**Operating system:**
OS X El Capitan Version 10.11.5 (15F34)

**Python version:**
2.7.10 (default, Oct 23 2015, 19:19:21) [GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)]

**What you did?**
Import PlexWatch DB

**What happened?**
Importing the first 10 entries and returning an error in the logs.

**What you expected?**
Import all entries.

**How can we reproduce your issue?**
Settings -> Extra Settings -> PlexWatch -> Import PlexWatch Database

**What are your (relevant) settings?**
Default settings 

**Link to logs:**
[plexpy.log.zip](https://github.com/drzoidberg33/plexpy/files/333327/plexpy.log.zip)

**Comment by JonnyWong16** [comment]
Looks like there might be a problem with your Plex Watch database:

```
2016-06-26 00:30:10 - ERROR   :: Thread-21 : Uncaught exception: Traceback (most recent call last):
  File "/Users/Bavo/Downloads/Test/plexpy/plexpy/logger.py", line 270, in new_run
    old_run(*args, **kwargs)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/Users/Bavo/Downloads/Test/plexpy/plexpy/plexwatch_import.py", line 308, in import_from_plexwatch
    for row in result:
DatabaseError: database disk image is malformed
```

Check #556 and #673

**Comment by bavocorteville** [comment]
Thanks, the fix in #556 solved the error. Nonetheless, the problem with the amount of entries is still there. (Importing 11 of 1237 entries now)

I'm importing from 'grouped' but i'm guessing it's processing only 11 entries from 'processed'

[plexpy.log.zip](https://github.com/drzoidberg33/plexpy/files/333505/plexpy.log.zip)

**Comment by JonnyWong16** [comment]
It looked like it imported successfully. Have you tried both "processed" and "grouped" and also changing the ignore interval?

**Comment by bavocorteville** [comment]
I tried "processed" and "grouped" with no interval, but no luck...

**Comment by JonnyWong16** [comment]
Send me your plexwatch.db file on Gitter or the forums.

**Comment by JonnyWong16** [comment]
Found the bug. It always imports the "processed" table no matter which one you select.

Fixed in the `dev` branch here: bc042fead7b4eea4eb6abab2012464b2adc679e0

You can make the same change in your local file `data/interfaces/default/app_import.html` or wait until I push it out to `master`. If you change it in your files, make sure you undo the change or you won't be able to auto-update.

**Comment by bavocorteville** [comment]
Thanks for the fix!
